version/version.go

```diff
@@ -21,7 +21,7 @@ const (
 var (
 	// P2PProtocol versions all p2p behavior and msgs.
 	// This includes proposer selection.
-	P2PProtocol uint64 = 8
+	P2PProtocol uint64 = 9
```

Comment:
According to @tychoish `UseLegacy` should be interoperable independent with false/true. I would rather clarify this first before trying to enforce this by bumping the version here.

Reply:
That makes sense.